doc/tutorials/setup_logical_partition_scenario_on_nuc7i7dnb.rst

```diff
@@ -0,0 +1,267 @@
+  root file systems via the SATA controller and the USB controller respectively.
+
+Update kernel image and modules of Pre-launched VM
+**************************************************
+#. On your development workstation, clone the ACRN kernel source tree, and build
+   the Linux kernel image that will be used to boot the privileged VMs:
+   
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-kernel.git
+      Cloning into 'acrn-kernel'... 
+      ...  
+      $ cd acrn-kernel 
+      $ cp kernel_config_uos .config 
+      $ make olddefconfig 
+      scripts/kconfig/conf  --olddefconfig Kconfig 
+      # 
+      # configuration written to .config 
+      # 
+      $ make 
+      $ make modules_install INSTALL_MOD_PATH=out/
+
+   The last two commands built the bootable kernel image in the
+   ``arch/x86/boot/bzImage``, and loadable kernel modules under the ``./out/``
+   folder.  Copy these files to a removable disk for installing on the NUC7i7DNB
+   board later.
+
+#. Current ACRN logical partition scenario implementation requires a multi-boot
+   capable bootloader to boot both ACRN hypervisor and the bootable kernel image
+   built from the previous step. You could install Ubuntu OS to the NUC7i7DNB
+   board on the NVMe SSD by following `this Ubuntu
+   tutorial <https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-desktop>`_.
+   The Ubuntu installer creates 3 disk partitions on the on-board NVMe SSD. By
+   default, the GRUB bootloader is installed on the ESP (EFI System Partition)
+   partition, which will be used to bootstrap ACRN hypervisor.
+#. After installing the Ubuntu OS, power off the NUC7i7DNB board, attach the
+   SATA disk and storage device with USB interface to the board. Power on the
+   board and make sure it boots the Ubuntu OS from the NVMe SSD, then copy the
+   loadable kernel modules built in Step 1 to the ``/lib/modules/`` folder on
+   both the mounted SATA disk and storage device with USB interface. For
+   example, assuming the SATA disk and storage device with USB interface are
+   assigned to ``/dev/sda`` and ``/dev/sdb`` respectively, the following
+   commands set up the partition mode loadable kernel modules onto the root file
+   systems to be loaded by the pre-launched VMs:
+
+   .. code-block:: none
+
+      # Mount the Clear Linux OS root filesystem on the SATA disk
+      $ sudo mount /dev/sda3 /mnt
+      $ sudo cp -r <kernel-modules-folder-built-in-step1>/lib/modules/* /mnt/lib/modules
+      $ sudo umount /mnt
+      # Mount the Clear Linux OS root filesystem on the USB flash disk
+      $ sudo mount /dev/sdb3 /mnt
+      $ sudo cp -r <path-to-kernel-module-folder-built-in-step1>/lib/modules/* /mnt/lib/modules
+      $ sudo umount /mnt
+
+#. Copy the bootable kernel image to the /boot directory:
+
+   .. code-block:: none
+
+      $ sudo cp <path-to-kernel-image-built-in-step1>/bzImage /boot/
+
+   .. note::
```

Comment:
These two notes can be combined and simplified.  At the beginning of this section you say to use a "development workstation" but here you're implying you can use your NUC as the development workstation?

```
.. note::
   Use ``cp`` as shown, if you're development workstation is the NUC, or ``scp`` if the image
   was built on another computer.
```